torchvision/prototype/transforms/functional/_utils.py

```diff
@@ -0,0 +1,43 @@
+from typing import List, Tuple, Union
+
+import PIL.Image
+import torch
+from torchvision.prototype import features
+from torchvision.transforms import functional_pil as _FP, functional_tensor as _FT
+
+get_dimensions_image_tensor = _FT.get_dimensions
+get_dimensions_image_pil = _FP.get_dimensions
+
+
+def get_chw(image: Union[PIL.Image.Image, torch.Tensor, features.Image]) -> Tuple[int, int, int]:
```

Comment:
Nit: I expected all these to be in `_meta` because they deal with meta-data stored in the Tensor.

Reply:
Can move.